Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/ActionsTreeView.cs

```diff
@@ -497,10 +579,9 @@ public static List<TreeViewItemData<ActionOrBindingData>> GetActionsAsTreeViewDa
                             if (isVisible)
                             {
                                 var name = GetHumanReadableCompositeName(nextBinding, state.selectedControlScheme, controlSchemes);
-                                var compositeBindingId = new Guid(nextBinding.id);
                                 compositeItems.Add(new TreeViewItemData<ActionOrBindingData>(GetIdForGuid(new Guid(nextBinding.id), idDictionary),
-                                    new ActionOrBindingData(false, name, actionMapIndex, false,
-                                        GetControlLayout(nextBinding.path), nextBinding.indexOfBinding)));
+                                    new ActionOrBindingData(isAction: false, name, actionMapIndex, isComposite: false,
```

Comment:
I think its great that you have added named arguments like this, it should likely be a new default style for us to use for any function. It makes it way easier to understand when inspecting code or reviewing.